toolchain/diagnostics/diagnostic.h

```diff
@@ -18,8 +18,12 @@
 namespace Carbon {
 
 enum class DiagnosticLevel : int8_t {
-  // A note, not indicating an error on its own, but possibly providing context
-  // for an error.
+  // Information about the location of another diagnostic, showing how we
+  // reached that location. This is currently only used for the "in import"
+  // message.
+  Context,
```

Comment:
We have a lot of things named "Context" in the toolchain, and the uses of this look like `, Context,`. What do you think of "PrefixNote", "Origin", or something like that?

Reply:
Renamed to `LocationInfo`, does that work better for you?

Reply:
Seems good enough.